src/modules/universal-discovery/components/search/css/search.component.css

```diff
@@ -45,7 +45,8 @@
 }
 
 .c-search__submit[disabled] {
-    opacity: 0.75;
+    opacity: 0.3;
+    cursor: not-allowed;
```

Comment:
Not: I slightly changed styles of:
![Screen Shot 2019-03-22 at 11 08 31](https://user-images.githubusercontent.com/10233057/54816691-34221e00-4c95-11e9-9728-ad9fff8ab506.png)
to match:
![Screen Shot 2019-03-22 at 11 08 41](https://user-images.githubusercontent.com/10233057/54816695-371d0e80-4c95-11e9-9ced-d0d0f0bdf1a4.png)
both in disabled state.